core/views.py

```diff
@@ -174,3 +174,7 @@ def coc_legacy(request, lang=None):
 #         'donor_list': donor_list,
 #         'quotes': DONOR_QUOTES,
 #     })
+
+
+def healthcheck(request):
```

Comment:
I wouldn't do this. The health check should check that the home page loads because that's the application we want to be healthy.

This view may return what you want while something in the application is broken and preventing the home page from rendering. Does that make sense?

Reply:
Yeah that makes sense, i'll change it to hit the homepage in healthcheck.sh and remove this view.